### PR TITLE
Update to Node 16

### LINKED ISSUE #83: Upgrade to node 16
GitHub Actions are requesting a migration of all actions from node12 to node16.
https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/

As a result we're receiving warnings in our workflows from this action:

```
Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: zaproxy/action-baseline
```

### PR DESCRIPTION
**Pull request by psiinon**
Also added a `check-run` action which runs the local baseline action against www.example.org

Example runs: https://github.com/psiinon/action-baseline/actions

More updates planned, but will submit them in different PRs...

Fix #83.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,3 +115,7 @@ This is currently only available with the `owasp/zap2docker-weekly` or `owasp/za
 See [https://github.com/zaproxy/zaproxy/tree/develop/zap/src/main/dist/lang](https://github.com/zaproxy/zaproxy/tree/develop/zap/src/main/dist/lang) for the full set of locales currently supported.
 
 You can help improve ZAP translations via [https://crowdin.com/project/owasp-zap](https://crowdin.com/project/owasp-zap). 
+
+## Build instructions
+
+Build using `npm run package`
```
Comment (thc202) [inline README.md:121]: IMO better add this to other doc, e.g. `BUILDING.md`. The `README.md` is published with the action and this is not relevant for its usage.

File: .github/workflows/check-dist.yml [hunk 2]
```diff
@@ -23,10 +23,10 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Set Node.js 12.x
+      - name: Set Node.js 16.x
```
Comment (thc202) [inline .github/workflows/check-dist.yml:26]: Remove the version.

File: .github/workflows/check-run.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    paths-ignore:
+      - '**.md'
+  workflow_dispatch:
+
+jobs:
+  check-run:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout # Required to test a local action
```
Comment (thc202) [inline .github/workflows/check-run.yml:19]: The comment could be removed (same for the other comment).

**Comment by psiinon** [comment]
Updated. 
The check-run is failing because of "Error: Resource not accessible by integration" - any suggestions for fixing? I think the check helps show the action actually runs, but having a check we know will always fail isnt ideal :/

**Comment by psiinon** [comment]
> No token specified so `allow_issue_writing` should be false? (Just guessing.)

Too obvious! That did the trick - thanks @kingthorin !

**Review by kingthorin — APPROVED** [review]

**Comment by psiinon** [comment]
Trying with a temp local website, dont expect it to work :/

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**